pkg/api/api.go

```diff
@@ -97,6 +97,7 @@ func (hs *HTTPServer) registerRoutes() {
 
 	r.Get("/d/:uid/:slug", reqSignedIn, redirectFromLegacyPanelEditURL, hs.Index)
 	r.Get("/d/:uid", reqSignedIn, redirectFromLegacyPanelEditURL, hs.Index)
+	r.Get("/g/*", reqSignedIn, hs.Index) // dashboard based on path
```

Comment:
g stands for? :)

Reply:
grafana?  any better suggestions?   In my mind everything will live under this eventually